data/json/items/comestibles/med.json

```diff
@@ -172,13 +172,82 @@
         "stack_size" : 9,
         "use_action" : {
             "type" : "heal",
-            "limb_power" : 3,
-            "head_power" : 1,
-            "torso_power" : 4,
+            "limb_power" : 0,
+            "bandages_power" : 4,
+            "bleed" : 0.9,
+            "move_cost" : 300
+        }
+    },
+    {
+        "id" : "bandages_makeshift",
+        "type" : "COMESTIBLE",
+        "comestible_type" : "MED",
+        "name" : "makeshift bandage",
+        "description" : "Simple cloth bandages.  Better than nothing.",
+        "weight" : 70,
+        "volume" : 1,
+        "price" : 600,
+        "material" : "cotton",
+        "symbol" : "!",
+        "color" : "white",
+        "charges" : 3,
+        "stack_size" : 9,
+        "use_action" : {
+            "type" : "heal",
+            "limb_power" : 0,
+            "bandages_power" : 2,
             "bleed" : 0.9,
             "move_cost" : 300
         }
     },
+  {
+    "id": "bandages_makeshift",
```

Comment:
Duplicate of `bandages_makeshift` above?

Reply:
Yes, it was.